### DISCUSSION TITLE
Open every unknown link with default browser

### DISCUSSION THREAD

**Discussion by BLACK4585** [LinkSheet/LinkSheet#486] [Q&A] [answered]
How can I enable this in the nightly builds? I think it was possible in the current stable build.
I basically want to open every link in the set Browser, which is not connected to any app.

**Comment by 1fexd** [comment]
> which is not connected to any app.

Connected as-in "if there is no app that can handle this link" or "if I haven't specifically set an app to always handle links on this host"?

If you are refering to the former, try `Auto-launch single browser` (requires `Improved intent resolver`)

**Reply by BLACK4585** [reply]
I mean the first one. When I select the preferred browser, it says that this browser is opened automatically when there is no native app found. Which seems not to work. It always shows me the bottom sheet.

**Reply by 1fexd** [reply, marked as answer]
Does enabling the experiments `Auto-launch single browser` and `Improved intent resolver` fix this issue?

**Reply by BLACK4585** [reply]
Ahh the first one did the trick. I thought I've tested it before, weird. 
Thank you for your help.
